backend/turip-admin/src/main/resources/templates/admin/pending-review.html

```
@@ -0,0 +1,282 @@
+
+<div class="container" style="margin-top: 80px;">
+    <div class="row">
+        <div class="col-lg-8">
+            <div class="admin-card">
+                <h3 class="section-title">1. 영상 및 기본 정보</h3>
+                <div class="mb-4">
+                    <h4 id="display_title" class="fw-bold mb-1" style="letter-spacing: -0.5px;">-</h4>
+                    <p id="display_channel" class="text-primary fw-bold small mb-3"></p>
+                </div>
+                <div id="video_preview" class="ratio ratio-16x9">
+                    <iframe id="video_iframe" src="" title="YouTube" allowfullscreen style="border: none;"></iframe>
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ iframe ] must have a value.

(src-not-empty)